frontend/Views/ViewDashboard.svelte

```diff
@@ -45,6 +82,26 @@
         });
     };
 
+    const calculateWidgetStatsKey = function (widget) {
+        return sha1((widget.settings.filter ?? []).join(""));
+    };
+
+    const filterViewForWidget = async function (widget) {
+        if (!widget.settings.filter) {
```

Comment:
isn't in js that anempty array (`filter=[]`) evaluates to `true` (not like in the python)?

Reply:
Fixed